packages/grafana-ui/src/components/index.ts

```diff
@@ -84,6 +84,7 @@ export {
   BigValueJustifyMode,
   BigValueTextMode,
 } from './BigValue/BigValue';
+export { Sparkline } from './Sparkline/Sparkline';
```

Comment:
Might be a good idea to tag the Sparkline component (in `Sparkline/Sparkline.tsx`) with `/** @internal */` to send the signal that we might have breaking changes outside of the regular release cycle.

Reply:
Good call, added